lib/ruleHelper.js

```diff
@@ -256,6 +269,22 @@ RuleHelper.prototype = {
         case "ArrowFunctionExpression":
             methodName = "";
             break;
+        case "AssignmentExpression":
+            if (RIGHT_OPERATORS.includes(node.operator)) {
+                methodName = this.normalizeMethodCall(node.right);
+            } else if (MATH_OPERATORS.includes(node.operator)) {
+                methodName = "";
+                break;
+            } else if (LOGICAL_OPERATOS.includes(node.operator)) {
+                // Issue #210: oh, no! we have two methods names we need to check, but this function
+                // may only return one. it's not immediately clear how to reconcile that with existing callers of normalizeMethodCall.
+                methodName = "";
+            }
+            else {
+                // this is the forcing function for us to complain about and implement support if new JS operators come up.
+                this.reportUnsupported(node, "Unexpected callable", `unexpected assignment with operator '${node.operator}' in normalizeMethodCall`);
```

Comment:
thought: I think that we may be able to cover this branch with a test case using that `parser: require.resolve("../parsers/fantasy-operator")` approach we used in some other test cases in the past for similar reasons.